CHANGELOG.asciidoc

```diff
@@ -2,6 +2,153 @@
 // each other on Github, so don't worry too much on using the right prefix.
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
+
+[[release-notes-8.4.2]]
+=== Beats version 8.4.2
+https://github.com/elastic/beats/compare/v8.4.1\...v8.4.2[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+
+*Auditbeat*
+
+
+*Filebeat*
+- [Httpjson]- Fix for pagination at root level not working when used with chaining.
```

Comment:
This one is missing a PR link. Can you add one from the relevant commit in the history? 

It also looks to be in the wrong section. This looks like a bug fix and not a breaking change.